src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/validation/FlowValidatorTest.java

```diff
@@ -199,6 +201,18 @@ public void shouldFailOnSpecifiedBothOuterVlansAndVlanStatistics() throws Invali
         flowValidator.checkFlowForCorrectOuterVlansWithVlanStatistics(flow);
     }
 
+    @Test(expected = InvalidFlowException.class)
+    public void shouldFailOnTooLowVlanStatistics() throws InvalidFlowException {
```

Comment:
It's funny but just today (on daily sync) we decided to do not use "should" in test names. 
Common pattern must be `findBlaBlaBlaTest` and `failOnTooLowVlanStatisticsTest`

Reply:
Ok, renamed.